feature/experimental/gribi/ate_tests/route_addition_during_failover_test/route_addition_during_failover_test.go

```diff
@@ -0,0 +1,667 @@
+				fluent.NextHopEntry().
+					WithNetworkInstance(*deviations.DefaultNetworkInstance).
+					WithIndex(index).
+					WithIPAddress(nextHops[i]).
+					WithElectionID(args.electionID.Low, args.electionID.High))
+
+			fluentNhgVar.AddNextHop(index, 64)
+		}
+		args.client.Modify().AddEntry(t, fluentNhgVar)
+	}
+
+	for ip := range virtualVIPs {
+		args.client.Modify().AddEntry(t,
+			fluent.IPv4Entry().
+				WithPrefix(virtualVIPs[ip]+"/32").
+				WithNetworkInstance(*deviations.DefaultNetworkInstance).
+				WithNextHopGroup(uint64(nhgID)).
+				WithElectionID(args.electionID.Low, args.electionID.High))
+	}
+
+	if err := awaitTimeout(args.ctx, args.client, t, time.Minute); err != nil {
+		t.Fatalf("Could not program entries via clientA, got err: %v", err)
+	}
+
+	if switchover {
+		for _, v := range args.client.Results(t) {
+			if v.ProgrammingResult == aftspb.AFTResult_FIB_PROGRAMMED {
+				fibProgrammedEntries = append(fibProgrammedEntries, v.Details.IPv4Prefix)
+			}
+		}
+	} else {
+		for ip := range virtualVIPs {
+			chk.HasResult(t, args.client.Results(t),
+				fluent.OperationResult().
+					WithIPv4Operation(virtualVIPs[ip]+"/32").
+					WithOperationType(constants.Add).
+					WithProgrammingResult(fluent.InstalledInFIB).
+					AsResult(),
+				chk.IgnoreOperationID(),
+			)
+		}
+
+		gr, err := args.client.Get().
+			WithNetworkInstance(*deviations.DefaultNetworkInstance).
+			WithAFT(fluent.IPv4).
+			Send()
+		if err != nil {
+			t.Fatalf("Error encountered during gRIBI Get operation: got: %v", err)
+		}
+
+		for ip := range virtualVIPs {
+			chk.GetResponseHasEntries(t, gr,
+				fluent.IPv4Entry().
+					WithNetworkInstance(*deviations.DefaultNetworkInstance).
+					WithNextHopGroup(uint64(nhgID)).
+					WithPrefix(virtualVIPs[ip]+"/32"),
+			)
+		}
+	}
+}
+
+// pushConfig pushes the configuration to the device using gNMI Replace.
+func pushConfig(t *testing.T, dut *ondatra.DUTDevice, dutPort *ondatra.Port, d *oc.Root) {
+	t.Helper()
+
+	intfName := dutPort.Name()
+	i := d.GetOrCreateInterface(intfName)
+	gnmi.Replace(t, dut, gnmi.OC().Interface(intfName).Config(), i)
+}
+
+// configureInterfaceDUT configures a single DUT layer 2 port.
+func configureInterfaceDUT(t *testing.T, dutPort *ondatra.Port, d *oc.Root, desc string) {
+	t.Helper()
+
+	i := d.GetOrCreateInterface(dutPort.Name())
+	i.Description = ygot.String(desc)
+	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	if *deviations.InterfaceEnabled {
+		i.Enabled = ygot.Bool(true)
+	}
+	t.Logf("DUT port %s configured", dutPort)
+}
+
+// generateSubIntfPair configures ATE/DUT SubInterfaces on the target device and returns
+// a slice of the corresponding ATE IPAddresses.
+func generateSubIntfPair(t *testing.T, dut *ondatra.DUTDevice, dutPort *ondatra.Port, ate *ondatra.ATEDevice, atePort *ondatra.Port, top *ondatra.ATETopology, d *oc.Root) []string {
+	t.Helper()
+
+	nextHops := []string{}
+	nextHopCount := 63 // nextHopCount specifies number of nextHop IPs needed.
```

Comment:
Bump this to 64 (to match the test description)?
63 worked with the previous indexing from 0

Reply:
updated, thank you